miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -82,7 +159,20 @@ class DeviceStatus(Enum):
     ManualSweeping = 13
 
 
-class DreameVacuumStatus(DeviceStatusContainer):
+class WaterFlow(Enum):
+    Unknown = -1
+    Low = 1
+    Medium = 2
+    High = 3
+
+
+class WaterTankStatus(Enum):
+    Unknown = -1
+    NotAttached = 0
+    Attached = 1
```

Comment:
I'd just use boolean here. Roborock is currently using `is_water_box_carriage_attached` which is not a great name for this, though.. :-P